app/controllers/buffered_articles_controller.rb

```diff
@@ -1,20 +1,20 @@
 class BufferedArticlesController < ApplicationController
   # No authorization required for entirely public controller
+
   def index
-    @article_urls = buffered_article_urls
-    render json: {
-      urls: @article_urls
-    }.to_json
+    render json: { urls: buffered_articles_urls }.to_json
   end
 
-  def buffered_article_urls
-    if Rails.env.production?
-      Article.
-        where("last_buffered > ? OR published_at > ?", 24.hours.ago, 20.minutes.ago).
-        select(:path).
-        map { |a| "https://#{ApplicationConfig['APP_DOMAIN']}#{a.path}" }
-    else
-      Article.all.map { |a| "https://#{ApplicationConfig['APP_DOMAIN']}#{a.path}" }
-    end
+  private
+
+  def buffered_articles_urls
+    relation = if Rails.env.production?
+                 Article.where("last_buffered > ? OR published_at > ?", 24.hours.ago, 20.minutes.ago)
+               else
+                 Article.all
+               end
+
+    paths = relation.pluck(:path)
+    paths.map { |path| "https://#{ApplicationConfig['APP_DOMAIN']}#{path}" }
```

Comment:
I wonder if we should add a helper for this at one point. A quick search shows various places throughout the app where we manually build paths like this.

Obviously not part of this PR, just thinking out loud.

Reply:
Agreed. There's also a `current_state_path` in the article decorator that doesn't feel right

Reply:
Cool, I added that to my personal todo list.